src/components/pages/ViewAllMeetings/ViewAllMeetings.js

```diff
@@ -0,0 +1,25 @@
+import React, { useState, useEffect } from 'react';
+import axiosWithAuth from '../../../utils/axiosWithAuth';
+import VAMData from './VAMData';
+
+const ViewAllMeetings = () => {
+  const [meetingData, setMeetingData] = useState([]);
+
+  useEffect(() => {
+    axiosWithAuth()
+      .get('/meetings')
+      .then(response => {
+        console.log(response);
```

Comment:
Why keep the log?  This code looks about ready to go.  A console.log in a GET might be overlooked by UI fixes.